chocolatey/plugins/modules/win_chocolatey_feature.ps1

```diff
@@ -24,12 +24,19 @@ param()
 $ErrorActionPreference = "Stop"
 
 # Documentation: https://docs.ansible.com/ansible/2.10/dev_guide/developing_modules_general_windows.html#windows-new-module-development
+$ValidStates = @("disabled", "enabled")
 function Get-ModuleSpec {
     @{
         options             = @{
             name  = @{ type = "str"; required = $true }
-            state = @{ type = "str"; default = "enabled"; choices = "disabled", "enabled" }
+            state = @{ type = "str"; default = "enabled"; choices = $ValidStates }
+            features = @{ type = "dict" }
```

Comment:
We already have a name parameter which is pretty standard across Ansible. Are we adding features here, so people can use name for one and features for multiple?

Reply:
Is there a precedent for this, as it seems really unintuitive?

Reply:
Yep, as I mentioned in the PR description this addition is heavily based on the pattern seen in [win_environment](https://docs.ansible.com/ansible/latest/collections/ansible/windows/win_environment_module.html).

Though in `win_environment` they have `name` and `value` _or_ the multiple-option dict `variables`.